File: Modules/HumanResource/PayrollSettings/PayrollSettings/TextFormatEpf.cs
using System.ComponentModel;
namespace HRMSoftware.PayrollSettings;
[Serenity.ComponentModel.EnumKey("HumanResource.PayrollSettings.TextFormatEpf")]
public enum TextFormatEpf
{
    [Description("CIMB")]
    CIMB = 1,
    [Description("KWSP")]
    KWSP = 2
}

File: Modules/ServerTypes/PayrollSettings/TextFormatEpf.ts
import { Decorators } from "@serenity-is/corelib";

export enum TextFormatEpf {
    CIMB = 1,
    KWSP = 2
}
Decorators.registerEnumType(TextFormatEpf, 'HRMSoftware.PayrollSettings.TextFormatEpf', 'HumanResource.PayrollSettings.TextFormatEpf');